inference/core/workflows/core_steps/sinks/roboflow/dataset_upload/v1.py

```diff
@@ -139,13 +139,13 @@ class BlockManifest(WorkflowBlockManifest):
         examples=[10, 60],
     )
     max_image_size: Tuple[int, int] = Field(
-        default=(512, 512),
+        default=(1920, 1080),
         description="Maximum size of the image to be registered - bigger images will be "
         "downsized preserving aspect ratio. Format of data: `(width, height)`",
-        examples=[(512, 512), (1920, 1080)],
+        examples=[(1920, 1080), (512, 512)],
     )
     compression_level: int = Field(
-        default=75,
+        default=100,
```

Comment:
When doing JPEG compression it seems `100%` might not be good value for default

```
Except for experimental purposes, never go above about Q 95; using Q 100
will produce a file two or three times as large as Q 95, but of hardly any
better quality.  Q 100 is a mathematical limit rather than a useful setting.
If you see a file made with Q 100, it's a pretty sure sign that the maker
didn't know what he/she was doing.
```
(copied from http://www.faqs.org/faqs/jpeg-faq/part1/section-5.html)

Reply:
Updated to 95